robocopkaka/hospital
# #161816914 Implement new flash messages

#### What does this PR do?
Add toast messages
#### Description of Task to be completed?
* Use Toastr to display flash messages
#### How should this be manually tested?
* Visit the app
* Signup or login
You should see a toast message at the top right corner of your page after completing your action
#### What are the relevant pivotal tracker stories?
[#161816914](https://www.pivotaltracker.com/story/show/161816914)
#### Any background context you want to add?
Displaying Flash messages with Semantic usually broke the UI
#### Screenshots
N/A
